docs/tutorials/neutron-data.ipynb

```diff
@@ -39,10 +39,18 @@
    "metadata": {},
    "outputs": [],
    "source": [
```

Comment:
Maybe as a follow up, but unless you anticipate your reader to be very familiar with Mantid from the offset, I would put more links in to mantid concepts for the various Workspace types, Loader etc discussed.